src/size.rs

```diff
@@ -2,10 +2,18 @@
 
 use super::command::Command;
 use display_interface::{DisplayError, WriteOnlyDataCommand};
-use generic_array::{
-    typenum::{U1024, U192, U360, U384, U512},
-    ArrayLength,
-};
+
+/// Workaround trait, since `Default` is only implemented to arrays up to 32 of size
+pub trait NewZeroed {
```

Comment:
This might be better as `pub(crate)` as AFAIK it's only used internally.

```suggestion
pub(crate) trait NewZeroed {
```

Reply:
Ah, ignore this I should've tested before commenting... doesn't work.

Reply:
We could sealed it, but not sure if worth it. Not sealing it might be useful if an user wants to write a buffer wrapper for some reason, although not sure if someone would want that...